app/utils/callback/tests/test_lava_callback.py

```diff
@@ -1,4 +1,4 @@
-# Copyright (C) Collabora Limited 2020
+# Copyright (C) Collabora Limited 2021
```

Comment:
Ah I believe this may be squashed with an earlier commit in this PR.

Reply:
squashed